Lib/test/support/__init__.py

```diff
@@ -2501,6 +2502,27 @@ def skip_unless_symlink(test):
     msg = "Requires functional symlink implementation"
     return test if ok else unittest.skip(msg)(test)
 
+_buggy_ucrt = None
+def skip_if_buggy_ucrt(test):
```

Comment:
Since this now lives in `test.support`, it might be worthwhile to mention 'time' in the name (`skip_if_buggy_ucrt_strfptime` or similar).

Reply:
I added strftime/strptime to the skip message and skip decorator name